Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2214 -- Subject Filter for Policy Repository

Resolves [EREGCSC-2214](https://jiraent.cms.gov/browse/EREGCSC-2214)

**Description:**

Add ability to filter Policy Documents by subject

**This pull request changes:**

- Adds Subject Filter to the Policy Respository page
- Adds Vue Router so URL query params control page state
- Adds basic styles from [this Figma comp](https://www.figma.com/file/KiWw729Nct6tY9gzmvEkVO/Subject-filter-only?node-id=108%3A1909&mode=dev)

**Steps to manually verify this change:**

1. Visit [policy repository page at experimental deployment](https://y65lx1o46e.execute-api.us-east-1.amazonaws.com/dev970/policy-repository) or build locally
2.  Use the subject selector.  Try using the cursor and the keyboard
3. Add and remove chips
4. Use the Back and Forward buttons
5. Select a bunch of chips and the copy and paste the URL into a new browser tab.  The same subjects should be selected on page load
